### PR TITLE
Kind instructions

### PR DESCRIPTION
**Pull request by Daniel-Sanche**
Fixes https://github.com/GoogleCloudPlatform/microservices-demo/issues/308:
- adds Kind to the list of supported local clusters
- improve instructions for accessing the local frontend

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,38 +99,52 @@ We offer the following installation methods:
...
    - [skaffold]( https://skaffold.dev/docs/install/) (ensure version ≥v0.20)
 
 1. Launch the local Kubernetes cluster with one of the following tools:
 
-    - Launch Minikube (tested with Ubuntu Linux). Please, ensure that the
+    - To launch **Minikube** (tested with Ubuntu Linux). Please, ensure that the
        local Kubernetes cluster has at least:
         - 4 CPU's
         - 4.0 GiB memory
+      ```shell
```
Comment (ahmetb) [inline README.md:115]: it's safer to add an empty line before this block

File: README.md [hunk 2]
```diff
@@ -98,38 +99,52 @@ We offer the following installation methods:
...
-    minikube start --cpus=4 --memory 4096
-    ```
-    
-    - Launch “Docker for Desktop” (tested with Mac/Windows). Go to Preferences:
+    - To launch **Docker for Desktop** (tested with Mac/Windows). Go to Preferences:
         - choose “Enable Kubernetes”,
         - set CPUs to at least 3, and Memory to at least 6.0 GiB
         - on the "Disk" tab, set at least 32 GB disk space
 
+    - to Launch a **Kind** cluster:
```
Comment (ahmetb) [inline README.md:124]: would it be worth adding the criteria above as well? (need a VM of this size etc) ?

Comment (Daniel-Sanche) [reply README.md:124]: I tried looking for this information, but kind doesn't seem to manage resources in the same way as the others. The [Istio docs](https://istio.io/docs/setup/platform-setup/kind/) don't mention anything about CPU/RAM either, even though it does for [minikube](https://istio.io/docs/setup/platform-setup/minikube/) and [docker for desktop](https://istio.io/docs/setup/platform-setup/docker/), so I decided to follow suit

Let me know if you have a better answer though

Comment (ahmetb) [reply README.md]: SGTM. we can skip.

**Pull request merged**